akka-docs-dev/rst/migration-guide-1.0-2.x.rst

```diff
@@ -159,7 +291,17 @@ Update procedure
 
 1. Replace all occurences of ``flatten(FlattenStrategy.concat)`` with ``flattenConcat()``
 
-TODO: code example
+Example
+^^^^^^^
+
+::
+
+   // This no longer works!
+   Flow[Source[Int, Any]].flatten(FlattenStrategy.concat)
+
+should be replaced by
+
+.. includecode:: /scala/code/docs/Migrations.scala#flatten
```

Comment:
Slightly misleading maybe? This change `flatten(FlattenStrategy.concat) with flattenConcat()` was only to Java DSL, right? because inference freaked out, in Scala we still have flatten?